src/roles/vsd-upgrade-complete/tasks/main.yml

```diff
@@ -86,6 +86,16 @@
 
   remote_user: "{{ vsd_default_username }}"
 
+- name: Check Certificates Expiration Time
+  shell: "{{ check_certificate_expiration_time_command }}"
+  register: certificate_expiration_time
+  remote_user: "{{ vsd_default_username }}"
+
+- name: Grep for ocspsigner user
+  assert:
+    that: certificate_expiration_time.stdout is search('ocspsigner')
+    fail_msg: "There is no ocspsigner user"
+
```

Comment:
We also need to validate that the certificate has not expired by comparing current time with the expiry timer. Also, once you have this running and tested, we need to test that this does not break 5.4.1 to 6.0.3 upgrades. Once the certificate validity is added, we also need to test for HA